apps/condo/domains/miniapp/gql.js

```diff
@@ -29,7 +29,7 @@ const SEND_B2C_APP_PUSH_MESSAGE_MUTATION = gql`
     }
 `
 
-const B2B_APP_NEWS_SHARING_CONFIG_FIELDS = `{ publishUrl previewUrl customFormUrl getRecipientsUrl icon { publicUrl } previewPicture { publicUrl } name ${COMMON_FIELDS} }`
+const B2B_APP_NEWS_SHARING_CONFIG_FIELDS = `{ publishUrl previewUrl customFormUrl getRecipientsUrl customGetRecipientsCountersUrl icon { publicUrl } previewPicture { publicUrl } name ${COMMON_FIELDS} }`
```

Comment:
strange naming. why not just `getRecipientsCunterUrl`?

Reply:
Done